cert/server.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
this is a snakeoil cert i generated so you can run things locally.

Reply:
That got my heart going for a bit.